ocs_ci/krkn_chaos/aggressive_clone_operations.py

```
@@ -0,0 +1,1001 @@
+        parent_pvc = parent_entry["pvc"]
+        parent_pvc.reload()
+        if parent_pvc.status != constants.STATUS_BOUND:
+            log.warning(
+                "Parent clone %s not bound, skipping nested clone", parent_pvc.name
+            )
+            return
+        self._create_and_track_clone(
+            parent_pvc, is_nested=True, original_source=parent_entry.get("source_pvc")
+        )
+
+    def _mount_verify_clones(self, workload_pvcs: List[PVC], cfg: Dict[str, Any]):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_mount_verify_clones`

Add return type annotation: `None`

(ANN202)